drivers/hmis/lib/form_data/records/client.json

```diff
@@ -55,7 +55,7 @@
           "link_id": "pronouns",
           "text": "Pronouns",
           "repeats": true,
-          "_TMP_IGNORED_field_name": "pronouns",
+          "field_name": "pronouns",
```

Comment:
FYI in case you haven't already done this -- every time you change a form definition you need to run `rails driver:hmis:seed_definitions` to reload it. After doing that, I can navigate to https://hmis.dev.test:5173/client/24440 on your branch, edit pronouns in the form, and have them successfully save and display on the card 🙌
And works with creating new clients as well: https://hmis.dev.test:5173/client/new